plugins/Table2/ui/Table2.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <v-data-table :headers="headers" :items="table" class="elevation-1" item-key='Component' :expand.sync="expanded" show-expand :item-class="itemClass">
+        <template v-slot:top>
+            <v-toolbar flat>
+                <v-toolbar-title>T.A.B.L.E.2.</v-toolbar-title>
+                <v-divider class="mx-4" inset vertical></v-divider>
+                <v-toolbar-description>{{ subtitle }}</v-toolbar-description>
+                <v-spacer></v-spacer>
+                <v-dialog v-model="dialog" max-width="500px">
+                    <template v-slot:activator="{ onn }"> <v-btn v-on="onn" color="primary" @click="initialize"> Reset </v-btn>  </template>
+                    <v-card>
+                        <v-card-title> <span class="text-h5">{{ formTitle }}</span> </v-card-title>
+                        <v-card-text> <v-container> <v-row> <v-textarea v-model="editedItem.Usage" label="Usage"></v-textarea> </v-row> </v-container> </v-card-text>
+                        <v-card-actions> <v-spacer></v-spacer> <v-btn color="blue darken-1" text @click="close"> Cancel </v-btn> <v-btn color="blue darken-1" text @click="save"> Save </v-btn> </v-card-actions>
+                    </v-card>
+                </v-dialog>
+            </v-toolbar>
+        </template>
+        <template v-slot:[`item.actions`]="{ item }"> <v-icon small class="mr-2" @click="editItem(item)"> mdi-pencil </v-icon> </template>
+        <template v-slot:[`item.Selected`]="{ item }"> <v-simple-checkbox @click="editItem(item); item.Selected = !item.Selected" v-model="item.Selected"></v-simple-checkbox> </template>
+        <template v-slot:expanded-item="{ headers, item }"> <td v-if="item.Details" :colspan="headers.length"> {{ item.Details }} </td> </template>
+        <template v-slot:[`item.data-table-expand`]="{ item, isExpanded, expand }">
+            <v-icon @click="expand(true)" v-if="item.Details && !isExpanded"> mdi-chevron-up </v-icon>
+            <v-icon @click="expand(false)" v-if="item.Details && isExpanded"> mdi-chevron-down </v-icon>
+        </template>
+    </v-data-table>
+</template>
+
+<style> .selected { color: gray } .unselected {} </style>
+
+<script>
+const { getCurrentBP } = await require('@bridge/env')
+const { readJSON } = await require('@bridge/fs')
+let configPath = `${getCurrentBP().replace(/\/BP\s*$/, "")}/config.json`
```

Comment:
You can use the `getCurrentProject()` method from the `@bridge/env` module instead. This allows you to get rid of of the `.replace(...)` call